### DISCUSSION TITLE
2023/9/12/gitbook-v2-intro

### DISCUSSION THREAD

**Discussion by giscus** [kaochenlong/kaochenlong-discuss#3] [General]
# 2023/9/12/gitbook-v2-intro

《為你自己學 Git（第二版）》是一本深入介紹 Git 版本控制系統的實用指南。無論你是新手還是有經驗的 Git 使用者，這本書都提供了清晰的解釋和示例，幫助你理解 Git 的核心概念和常用指令。作者以淺顯易懂的方式解釋了 Git 的運作原理，並分享了在多種情境下如何使用 Git

https://kaochenlong.com/2023/9/12/gitbook-v2-intro.html

**Comment by wasp33** [comment]
什麼時候出第二版（敲碗）

**Reply by kaochenlong** [reply]
原定是 2024 年的第一季，但又被別的事卡住，請再等我一下了 :)
